tensorflow/lite/micro/kernels/micro_ops.h

```diff
@@ -54,6 +54,7 @@ TfLiteRegistration Register_CONCATENATION();
 TfLiteRegistration Register_COS();
 TfLiteRegistration Register_DEQUANTIZE();
 TfLiteRegistration Register_EQUAL();
+TfLiteRegistration Register_EXP();
```

Comment:
move to the tflite namespace section up top.

Reply:
fixed.